### ISSUE TITLE
App is completely inaccessible with screenreader

### ISSUE THREAD

**Issue by heull001** [etesync/ios#103]
Unfortunately the app is absolutely unusable for screenreader-users. Here the main problems:

- I don't know how to sync, there is no button or anything else I could use, so I am not able to sync
- the button on top of the app (main menu, back ...) is not reachable with swiping to last or next element
- not every button is marked as such, e.g. "my calendar" on main screen
- no idea what the button "Journal menu" should do, but if I click, it happens nothing
- headlines should be marked as such, e. g. "Adress books" on main screen

If you wish to test, you can enable the builtin screenreader voiceover via settings -> accessibility -> voiceover.

I recomment a look on <https://developer.apple.com/documentation/accessibility/supporting_voiceover_in_your_app/>. It gives usefull hints for easy increasing accessibility.

If you have any questions, don't hesitate to contact me. I am completely willing to help. Unfortunately I am not able to help coding by myself, I don't have the infrastructure to build iOS apps.

**Comment by tasn** [comment]
Interesting, I actually tested it quite a bit with screenreaders, I wonder what has changed!

**Comment by heull001** [comment]
Of course, I don't know when you tested, but I looked at Etesync back in November 2020, but then didn't pursue it for other reasons. And even then, the usability was similarly bad, but I can't say for sure if it was that bad.
